lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/TaskManager.cs

```diff
@@ -89,6 +89,11 @@ internal sealed class TaskManager
         private int _numberOfAppErrors = 0;
 
         /// <summary>
+        /// Indicate if master task is completed running properly
+        /// </summary>
+        private bool _masterTaskCompletedRunning = false;
```

Comment:
I suggest something like `_masterTaskCompletedCalculations` or `_masterTaskProducedResults`. Current name is pretty much the same as `_masterTaskCompleted` :-)